lib/api/v1/controllers/auth.go

```diff
@@ -0,0 +1,34 @@
+package controllers
+
+import (
+	"net/http"
+
+	errorhelpers "github.com/markgravity/golang-ic/helpers/error"
+	jsonhelpers "github.com/markgravity/golang-ic/helpers/json"
+	"github.com/markgravity/golang-ic/lib/api/v1/forms"
+
+	"github.com/gin-gonic/gin"
+)
+
+type AuthController struct {
+	BaseController
+}
+
+func (AuthController) SignUp(ctx *gin.Context) {
+	form := forms.SignUpForm{}
+
+	err := ctx.ShouldBindJSON(&form)
+	if err != nil {
+		jsonhelpers.RenderErrorWithDefaultCode(ctx, http.StatusBadRequest, err)
+		return
+	}
+
+	_, err = form.Save()
+	if err != nil {
+		statusCode := errorhelpers.GetErrorStatusCode(err)
+		jsonhelpers.RenderErrorWithDefaultCode(ctx, statusCode, err)
```

Comment:
I see there is already a list of error codes in the JSON helper. 
Wouldn't is be more explicit and straight forward if we had a method like:

```suggestion
		jsonhelpers.RenderUnprocessableEntityError(ctx, err)
```

According to the controller tests, there is only this error possible (or at least, tested ^^).